src/main/xcontainers/BodyItemsContainer.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Why?

Reply:
If you don't want to see eslint in your text editor, you can just press `⌘⇧p` and type `Disable ESLint`

That said, I'd like to create some kind of linting convention once we get past MVP presentation

Reply:
kk, I'll remove that 👍